From 3b9e7913a6d14cb09ed91ce8a1110955bda200ef Mon Sep 17 00:00:00 2001
From: Alexandre Pretyman <alexandre.pretyman@gmail.com>
Date: Tue, 14 Jul 2015 09:12:38 -0400
Subject: [PATCH] Change configure.ac from OpenWrt to support Mac OS

This change is from the OpenWrt project, from the URL:
https://dev.openwrt.org/ticket/18998
It enables patchelf to sucessfully compile and run under Mac OS and
keeps compatibility with Linux.

Tested under Mac OS X 10.9.5 and Ubuntu 15.04 to patch a binary of type
ELF 32-bit LSB executable - the patch fully worked

This patch is credited to Felix Fietkau
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 8ca9ad4..69e12de 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1,4 +1,4 @@
-AC_INIT([patchelf], m4_esyscmd([echo -n $(cat ./version)]))
+AC_INIT([patchelf], m4_esyscmd([printf $(cat ./version)]))
 AC_CONFIG_SRCDIR([src/patchelf.cc])
 AC_CONFIG_AUX_DIR([build-aux])
 AM_INIT_AUTOMAKE([-Wall -Werror dist-bzip2 foreign color-tests parallel-tests])
